pom.xml

```diff
@@ -1579,17 +1580,6 @@
         <version>${scala.version}</version>
       </dependency>
 
-      <dependency>
```

Comment:
It needs to be removed to support both Kafka2 and Kafka3 version-based streaming. In the future, when we use only Kafka3, we can bring the Kafka client dependency here.

Reply:
Suggest keeping it as is and only override the version in kafka3 module. This way we can ensure existing modules are unchanged

Reply:
reverted. Thanks